src/components/views/elements/FacePile.tsx

```diff
@@ -65,7 +61,7 @@ const FacePile: FC<IProps> = ({
     );
 
     const content = (
-        <AccessibleButton className="mx_FacePile" onClick={props.onClick ?? null}>
+        <AccessibleButton {...props} className="mx_FacePile" onClick={onClick ?? null}>
```

Comment:
is the `?? null` required?

Reply:
Yes because `undefined` != `null`, AccessibleButton does not accept `undefined`